gradle.properties

```diff
@@ -77,6 +77,7 @@ kotlinVersion=1.7.20
 snakeYamlVersion=1.33
 jdomVersion=2.0.6.1
 woodstoxVersion=6.4.0
+commonsNetVersion=3.9.0
```

Comment:
Why are we adding this new dependency?

Reply:
> Why are we adding this new dependency?
commons-net comes as part of hadoop, and since we are not upgrading hadoop, so we are enforcing the upgrade for commons-net library. hence added here.